src/lib/formassembly/iframe-resize-helper.js

```diff
@@ -0,0 +1,201 @@
+
+// Source: https://tfaforms.com/js/iframe_resize_helper.js
+/*
+ * Helper script for iframe-published FormAssembly.com forms.
+ * Auto-resize iframe to fit content.
+ */
+function resizeHelper() {
+    /*
+     * A backwards compatible implementation of postMessage
+     * by Josh Fraser (joshfraser.com)
+     * released aunder the Apache 2.0 license.
```

Comment:
It seems like we're good to use code under Apache 2.0, but I thought it would be good to confirm, have we used code under this license before? I'm not seeing references in the `ui` repo.

Reply:
Good call out 🤔